### DISCUSSION TITLE
Default configuration files for LWC and AURA linters

### DISCUSSION THREAD

**Discussion by MouadRabihi07** [oxsecurity/megalinter#3425] [Q&A] [answered]
Greetings, i'm using mega-linter with Salesforce flavor, and i'm struggling to find the configuration files that are used by default by the sfdx-scanner-lwc and sfdx-scanner-aura. I want to use them as templates to customize further. 
PS : they don't exist in the TEMPLATES folder.

**Comment by nvuillam** [comment, marked as answer]
@MouadRabihi07 I removed them because they were messing with the local configuration defined in the sfdx project

As sfdx-scanner is more and more recommended by SF, I wanted to make it also possible to run the same rules locally (and that wouldn't work if these rules are taken from MegaLinter default ones)

In your aura and lwc folders, there should be a .eslintrc.json, that will be used by sfdx-scanner

If you don't have them, you should be able to get them by calling `sfdx force:project:create --projectname` in any folder then get the .eslintrc.json files from the generated files

**Reply by MouadRabihi07** [reply]
Thanks for the response. I see that the RECOMMENDED plugin is used by default.

**Reply by nvuillam** [reply]
yes, let's follow Salesforce recommendations ;)

**Reply by MouadRabihi07** [reply]
Hello again Nicolas. So i tried tweaking the .eslintrc.json file that's inside the LWC folder, but the same rules are applied, which makes me think again that the linter calls the config file from another place seeing that the changes to the file are not applied. what do you think ?

**Reply by nvuillam** [reply]
I am currently facing the same issue with a project of mine, so keep your eyes around here , I'll know more soon ^^ ( sfdx-scanner is not the most easy to use tool, but it is SF's standard so we'll have to find a way to efficiently use it anyway :) )
